projects/plugins/jetpack/extensions/blocks/map/editor.scss

```diff
@@ -36,3 +36,14 @@
 .wp-block-jetpack-map__height_input {
 	display: block;
 }
+
+.component__add-point__popover {
+	.components-popover__content {
+		width: 150px;
```

Comment:
I noticed that the placeholder text gets cut off at this width in German:

![image](https://user-images.githubusercontent.com/14988353/110744045-281e3500-828d-11eb-971e-f7dc6df59d7b.png)

It'd also be good to add a bit more width so that you can better see what you're typing if you're choosing a place that is a bit longer, like if you type in "National Gallery of Australia". How about `200px`?

Reply:
Updated to 250px, this looks a lot better :+1:

<img width="352" alt="Screen Shot 2021-03-15 at 10 50 19 AM" src="https://user-images.githubusercontent.com/1464705/111198164-80e71980-857c-11eb-881f-4848d0b9343b.png">